Readme.md

```diff
@@ -27,78 +41,73 @@ sudo python3 -m pip install tensorflow==2.4.1 -f https://tf.kmtea.eu/whl/stable.
+
+
+![dpusc-output](doc/fig/dpusc-output.png)
+
+## Example-Cifar10 Resnet18 DEMO
+
+> Now only support IMAGE mode
+
+After selecting the first application "Cifar10_Resnet18_DEMO" and choosing the input and output formats, you will get the following results and find the output file in the folder.
+
 ![cnn-result-02](doc/fig/cnn-result-02.png)
 
-## Example - YOLO
-```bash   
-# Inference with image, output DP, using yolo
-python3 dpusc -i dataset/images/moto.jpg -x yolo -t dp
-```
-After execute above command(use YOLO xmodel), you will get the result and the output image like below:
-![yolo-result-01](doc/fig/yolo-result-01.png)
-![yolo-result-02](doc/fig/yolo-result-02.png)
 
+## Example-Object detection
 
-## Example - LPR
-```bash   
-# Inference with image, output image, using LPR
-python3 dpusc -i dataset/images/lpr-2.jpg -x yolo -lpr -t image
-```
-After execute above command(use LPR mode), you will get the result and the output image like below:
-![lpr-result-01](doc/fig/lpr-result-01.png)
-![lpr-result-02](doc/fig/lpr-result-02.png)
+After selecting the second application "Object detection" and choosing the input and output formats, you will get the following results and find the output file in the folder in the format you selected. 
 
-## Other Options
-```bash 
-# Inference with video, output image, using yolo
-python3 dpusc -v dataset/videos/walking_humans.nv12.1920x1080.h264 -x yolo -t image
+Additionally, during inference, you can view inference information, including bounding boxes and FPS, in the terminal window.
 
-# Inference with video, output video, using yolo
-python3 dpusc -v dataset/videos/walking_humans.nv12.1920x1080.h264 -x yolo -t video
+![yolo-result-01](doc/fig/yolo-result-01.png)
+![yolo-result-02](doc/fig/yolo-result-02.png)
 
-# Inference with webcam, output DP, using yolo
-python3 dpusc -c 0 -x yolo -t dp
 
-# Inference with webcam, output image, using yolo
-python3 dpusc -c 0 -x yolo -t image
+## Example-Automatic number-plate recognition
 
-# Inference with video, output DP, using LPR
-python3 dpusc -v <video path> -x yolo -lpr -t dp
+After selecting the third application "Automatic number-plate recognition" and choosing the input and output formats, you will get the following results and find the output file in the folder in the format you selected. 
 
-# Inference with video, output Video, using LPR
-python3 dpusc -v <video path> -x yolo -lpr -t video
+Additionally, during inference, you can view inference information, including bounding boxes, FPS, and license plate detection results, in the terminal window.
+
+![lpr-result-01](doc/fig/lpr-result-01.png)
+![lpr-result-02](doc/fig/lpr-result-02.png)
 
-```
 
-# Dataset rules
-### If run with CNN, you must follow the format of dataset naming rule which is label on the prefix of file name.  
-e.g.   
-  - at images_demo we detect cat or dog.  
-  - at images_usb we detect perfect or defect.  
-  
-and so on.  
 
 # Config.json
```

Comment:
We should keep the structure of README's section name style. E.g. How to modify the Config.json.